aikau/src/test/resources/alfresco/layout/AlfSideBarContainerTest.js

```diff
@@ -61,7 +61,7 @@ define(["intern!object",
       },
 
       "Test Sidebar Placement": function () {
-         return browser.findByCssSelector(".alfresco-layout-AlfSideBarContainer .sidebar #SIDEBAR_LOGO")
+         return browser.findByCssSelector(".alfresco-layout-AlfSideBarContainer .alfresco-layout-AlfSideBarContainer__sidebar #SIDEBAR_LOGO")
```

Comment:
Because of BEM, there's no need to nest rules, so we should be able to get rid of the parent `.alfresco-layout-AlfSideBarContainer` in all of the examples in this file.